src/llfs/bloom_filter.test.cpp

```diff
@@ -149,10 +149,19 @@ TEST(BloomFilterTest, RandomItems)
   }
 
   for (const auto& s : stats) {
-    EXPECT_LT(s.second.actual_rate() / s.second.expected_rate, 1.01)
+    const u64 word_count = s.first.first;
+    const u16 hash_count = s.first.second;
+    const double error_rate = s.second.actual_rate();
+
+    EXPECT_LT(s.second.actual_rate() / s.second.expected_rate, 1.02)
```

Comment:
nit: could use `error_rate` defined above in place of all usages of `s.second.actual_rate()` below, and could do the same by creating a new variable for with `s.second.expected_rate`.

Reply:
Fixed: added `actual_fpr` and `expected_fpr` local variables and used them in place of the expressions they derive from in this loop.